features/step_definitions/file_steps.coffee

```diff
@@ -5,17 +5,21 @@ path = require 'path'
 module.exports = ->
 
   @Given /^I have no test files$/, (done) ->
-    done() # No op for readability
+    done()
 
 
-  @Given /^I have 1 (passing|failing) test$/, (status, done) ->
-    outputTestFile path.join(@tmpDir, 'example_spec.coffee'), passing: status is 'passing', done
+  @Given /^I have a (passing|failing) test$/, (status, done) ->
+    outputTestFile path.join(@tmpDir, 'example_spec.coffee'),
+                   passing: status is 'passing',
+                   done
```

Comment:
In terms of `reserve left-aligned argument blocks for method calls with named arguments`: What about when they're are multiple arguments where the first has named arguments?
```coffee
createUser 
  firstname: 'Charlie'
  lastname: 'Rudolph'
  done

# vs

createUser firstname: 'Charlie', lastname: 'Rudolph'
           done

# vs

createUser 
  firstname: 'Charlie', lastname: 'Rudolph'
  done
```
---
In response to `People get used to expecting each argument on a separate line, and form habits.`. The habit is broken due to objects, but coffeescript has special treatment of objects (I made a PR yesterday to cson-parser saw this difference) so they need to be looked at in a different light

---
In response to that exclamation, `I'm getting the parameter, but somehow half of the data is missing!`, I think thats brought on by coffeescript. If there were no implicit braces, that wouldn't happen (we can turn on a coffeelint rule for no implicit braces). This can definitely happen when things are aligned to the right as well
```coffee
createUser firstname: 'Charlie',
           lastname: 'Rudolph'
```